docs/conf.py

```diff
@@ -35,7 +35,7 @@
 import sphinx.environment
 from docutils.utils import get_source_line
 
-def _warn_node(self, msg, node):
+def _warn_node(self, msg, node, **kwargs):
     if not msg.startswith('nonlocal image URI found:'):
         self._warnfunc(msg, '%s:%s' % get_source_line(node))
 
```

Comment:
I was guessing that's what all the Travis failures are about. Maybe they aren't related? In any case we can't merge until Travis passes ;)